src/SFA.DAS.EAS.Portal/Configuration/ConfigurationExtensions.cs

```diff
@@ -17,5 +17,11 @@ public static TConfiguration GetPortalSection<TConfiguration>(this IConfiguratio
         {
             return configuration.GetPortalSection(subSectionPaths).Get<TConfiguration>();
         }
+
+        public static TConfiguration GetSection<TConfiguration>(this IConfiguration configuration, string key,   params string[] subSectionPaths)
```

Comment:
It'd be nice if we changed the existing `GetPortalSection` methods to use this new method. (We'd have to introduce a new `GetPortal` that returned `IConfigurationSection` and accepted a key.)